README.md

```diff
@@ -109,3 +71,45 @@ The following files will be created:
 - `index.html`: start page for the project
 - `pyscript.toml`: project metadata and config file
 - `main.py`: a "Hello world" python starter module
+
+#### Use --wrap to embed a python file OR a command string
+
+- ##### Embed a Python script into a PyScript HTML file
+
+```shell
+$ pyscript create --wrap <filename.py>
+```
+
+This will generate a project i.e. a new directory named `filename` under the current directory.
+
+Similar to the above, interactive prompts will further ask for metadata information.
+
+The following files will be created:
+- `filename.html`: embeds the code of `filename.py` in a `<py-script>` tag.
+- `pyscript.toml`: project metadata and config file
+
+This can be overridden with the `-o` or `--output` option:
+
+```shell
+$ pyscript create --wrap <filename.py> -o <another_filename.html>
```

Comment:
See my comment above. I wouldn't  do this since I think it just adds complications. If it's something users ask in the future we can get back to it...

Also, we wouldn't create a `py` file? Would the code be inline?

Reply:
Yes, `wrap` assumes that the code is inline. Similar to the comment above.

It is basically borrowing the behaviour of `wrap` from the past aka `inline` code and blending it with the behaviour of creating a project folder for it.

Reply:
Done!